src/js/modules/hasFontLoaded/index.js

```diff
@@ -0,0 +1,27 @@
+import FontFaceObserver from 'fontfaceobserver';
+import { logError } from '@justeat/f-logger';
+
+/**
+ * @overview hasFontLoaded reduces the amount of time a user has invisible text when using webfonts.
+ *
+ * @module hasFontLoaded
+ */
+
+/**
+* Init method initialises the FontFaceObserver events
+*
+*/
+
+export const hasFontLoaded = () => {
```

Comment:
Would this be better named `initLoadFont`?

Reply:
It doesn't actually load the font – it just provides a way of observing if the font has loaded.

I'm not entirely sure if we even need this method anymore tbh, as this was mostly to monitor the loading of the font, whereas the way that we're recommending the font be included should block render via preload. I think I'll remove this and then if it's needed when we roll out, can look at it again in future.

Reply:
Have now removed this